contents/docs/product-analytics/ai.md

```diff
@@ -0,0 +1,41 @@
+---
+title: AI and LLM insights (beta)
```

Comment:
Langfuse doesn't use "AI" once on term homepage, so lets not as well. I think it is a bit of a buzzword, just LLM is fine.
```suggestion
title: LLM insights (beta)
```